google/cloud/spanner/integration_tests/database_admin_integration_test.cc

```diff
@@ -165,6 +169,103 @@ TEST(DatabaseAdminClient, DatabaseBasicCRUD) {
   db_list = get_current_databases();
   ASSERT_EQ(1, std::count(db_list.begin(), db_list.end(), db.FullName()));
 
+  // Tests for Backup are taking long time. To run them, set
+  // RUN_SLOW_INTEGRATION_TESTS environment variable to yes.
+  if (run_slow_integration_tests == "yes") {
+    auto backup_future = client.CreateBackup(
```

Comment:
You probably want to check if these need a `!emulator` guard.

Reply:
Thanks Added the guard.